src/ReceivingList/ReceivingListFilter/ReceivingListFilter.js

```diff
@@ -96,6 +90,22 @@ const ReceivingListFilter = ({
         name={FILTERS.LOCATION}
         onChange={adaptedApplyFilters}
       />
+
+      <AcqCheckboxFilter
+        activeFilters={activeFilters[FILTERS.RECEIVING_STATUS]}
+        labelId="ui-receiving.filter.receivingStatus"
+        name={FILTERS.RECEIVING_STATUS}
+        onChange={adaptedApplyFilters}
+        options={PIECE_STATUS_OPTIONS}
+      />
+
+      <AcqUnitFilter
+        id={FILTERS.ACQUISITIONS_UNIT}
```

Comment:
I would say id's are better to be unique, so maybe it's a good idea to make them more complex, something like `id={`filter-${FILTERS.ACQUISITIONS_UNIT}`}

Reply:
correct, to avoid accessibility issues